fearless_simd/examples/unpremul.rs

```diff
@@ -1,12 +1,12 @@
 // Copyright 2025 the Fearless_SIMD Authors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
-use fearless_simd::{Level, Simd};
+use fearless_simd::Level;
 
```

Comment:
This needs a TODO comment of some kind. I'd also probably prefer these lines were of the form:
```rust
#[cfg(target_arch= ...)]
 // TODO: Our f16 support doesn't actually exist yet, but this is what it will look like when it does.
#[cfg(any())]
```

However, because this PR is special, I'll leave this unaddressed for now.